ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -151,7 +152,7 @@ open class PopupMenuController: DrawerController {
         return view
     }()
 
-    private var separator: Separator?
+    private lazy var divider: MSFDivider = .init()
```

Comment:
why sometimes we are var foo = MSDivider() and seomtimes var divider: MSFDivider = .init()?

Reply:
Good question. I assume I generally tried to match the previous style of the Separator as closely as I could. Though this one is a bit more a change, also going from optional to lazy.

Reply:
One difference could be that member variables always have to be `var divider: MSFDivider` due to a swiftlint rule, while we generally avoid declaring types explicitly inside method bodies.